### PR TITLE
topkg-windows.1.0.7

### PR DESCRIPTION
**Pull request by Chimrod**
Update to latest topkg

### CODE REVIEW

**Comment by Chimrod** [comment]
Hello,

I’ll need some help in order to understand why the test failed. I will be able to submit new package, but I have to be able to reproduce the same error locally in order to submit correct PR.

Thanks !

**Comment by toots** [comment]
@Chimrod you can run the tests locally with a verbose mode:
```shell
% VERBOSE=1 ./tests/run_tests.sh
```

**Comment by Chimrod** [comment]
I have this log after some time:

```
Step 10/12 : RUN cd /usr/src/mxe/ && cat /home/opam/mxe-deps | xargs make
 ---> Running in 9f20b178d153
[pkg-list]  cc cmake
[download]    binutils-2.36.1.tar.bz2
[download]    ccache-3.6.tar.xz
[download]    mingw-w64-v9.0.0.tar.bz2
[download]    gmp-6.1.2.tar.xz
[download]    isl-0.16.1.tar.bz2
MXE Warning! Downloading isl from second URL.
MXE Warning! Downloading isl from backup.

Download failed or wrong checksum of package isl!
------------------------------------------------------------
Connecting to d1yihgixbnrglp.cloudfront.net (d1yihgixbnrglp.cloudfront.net)|13.32.158.139|:80... failed: No route to host.
Connecting to d1yihgixbnrglp.cloudfront.net (d1yihgixbnrglp.cloudfront.net)|13.32.158.212|:80... failed: No route to host.
--2023-10-02 20:51:31--  http://d1yihgixbnrglp.cloudfront.net/isl%2D0%2E16%2E1%2Etar%2Ebz2_412538bb65c799ac98e17e8cfcdacbb257a57362acfaaff254b0fcae970126d2
Connecting to d1yihgixbnrglp.cloudfront.net (d1yihgixbnrglp.cloudfront.net)|13.32.158.181|:80... failed: No route to host.
Connecting to d1yihgixbnrglp.cloudfront.net (d1yihgixbnrglp.cloudfront.net)|13.32.158.26|:80... failed: No route to host.
Connecting to d1yihgixbnrglp.cloudfront.net (d1yihgixbnrglp.cloudfront.net)|13.32.158.139|:80... failed: No route to host.
Connecting to d1yihgixbnrglp.cloudfront.net (d1yihgixbnrglp.cloudfront.net)|13.32.158.212|:80... failed: No route to host.
Download failed!
------------------------------------------------------------
[log]      /usr/src/mxe/log/isl-download

make: *** [Makefile:707: download-only-isl-0.16.1.tar.bz2] Error 1
[failed]🚫
```

I have no idea how to interpret this!

**Comment by Chimrod** [comment]
I do not understand why the tests are failing. It looks like there is a timeout after a while, but I do not know the test process…

**Comment by toots** [comment]
I have no idea either unfortunately. This looks like a CI issue but the tests are passing with other packages. I'll see if I can find the time to look at it at some point soon.

File: packages/topkg-windows/topkg-windows.1.0.7/opam [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+authors: "The topkg programmers"
+license: "ISC"
+tags: ["packaging" "ocamlbuild" "org:erratique"]
+homepage: "https://erratique.ch/software/topkg"
+doc: "https://erratique.ch/software/topkg/doc"
+bug-reports: "https://github.com/dbuenzli/topkg/issues"
+depends: [
+  "ocaml" {>= "4.05.0"}
+  "ocaml-windows" {>= "4.05.0"}
+  "ocamlfind" {build & >= "1.6.1"}
```
Comment (toots) [inline packages/topkg-windows/topkg-windows.1.0.7/opam:39]:
```suggestion
  "ocamlfind" {build & >= "1.6.1"}
  "opam-installer"
```

**Pull request merged**